src/main/java/racingcar/domain/RandomStrategy.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain;
+
+import java.util.Random;
+
+public class RandomStrategy implements RacingStrategy{
+
+    private static final int MAXVALUE = 10;
```

Comment:
```suggestion
    private static final int MAX_VALUE = 10;
```

단어 중간에 언더바를 사용해서 띄어쓰기로 구분을 해주시면 가독성이 더 좋아집니다.